src/server.rs

```diff
@@ -324,8 +324,8 @@ impl DistClientContainer {
             }};
         }
         // TODO: NLL would avoid this clone
```

Comment:
Probably this comment should've been removed?

Reply:
Doh.  Yes, will fix.